public/app/features/profile/ReactProfileWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import { UserProvider } from 'app/core/utils/UserProvider';
+import { UserProfileEditForm } from './UserProfileEditForm';
+import { SharedPreferences } from 'app/core/components/SharedPreferences/SharedPreferences';
+
+export class ReactProfileWrapper extends PureComponent {
```

Comment:
You don't need class component here, it's only render so it can be a function component

Reply:
Also, ReactProfileWrapper sounds weird to me.. What about simply `UserProfile` / `UserProfileEditor`?

Reply:
It's only here so that this can be converted it pieces. The name isn't important.